.travis.yml

```diff
@@ -18,6 +21,21 @@ script:
   - sbt -DscalaJSStage=full ++$TRAVIS_SCALA_VERSION validateCode test mimaReportBinaryIssues publishLocal
   - sbt ++$TRAVIS_SCALA_VERSION docs/test docs/validateDocs
 
+jobs:
+  include:
+    - stage: publish
+      name: "TEMPORARY, will become Publish artifacts to Bintray"
+      script: sbt +mimaReportBinaryIssues
+      scala: 2.13.1
```

Comment:
why do we need the scala version?

Reply:
Otherwise the Travis Matrix kicks in and runs it for 2.12 and 2.13.

Reply:
I think you lost me? Are you saying that the `scala` section on line 13 will cause it to run twice?

I think we should remove this. The build.sbt already define the default scala version and the the `+` will take care of the cross building. 

Btw, play-soap doesn't have it: https://github.com/playframework/play-soap/blob/master/.travis.yml

Reply:
We don't have a definition for state `test` in this file? Is that a default staget? And will it pick what is defined in `script` (line 20)

Reply:
Play SOAP doesn't use a Travis Matrix.

Yes, `test` automatically inherits the global `script` as it doesn't have its own.

Reply:
That was my point. I was suggesting to remove the mix of matrix with stages.  Let's only have stages, otherwise we need extra brain cycles to map the effect they have on each other.

Reply:
I unrolled it now.